generated/python/gapic-google-cloud-datastore-v1/setup.py

```diff
@@ -9,15 +9,15 @@
 import sys
 
 install_requires = [
-    'googleapis-common-protos>=1.3.5, <2.0.0dev',
-    'google-gax>=0.14.1, <0.15.0dev',
-    'grpc-google-datastore-v1>=0.11.1, <0.12.0dev',
-    'oauth2client>=2.0.0, <4.0.0dev',
+    'googleapis-common-protos>=1.5.0, <2.0dev',
+    'google-gax>=0.14.1, <0.15dev',
```

Comment:
@bjwatson If `google-gax==0.15` just went out today, why doesn't this version depend on it?